jwst/saturation/saturation.py

```diff
@@ -84,7 +93,7 @@ def flag_saturation(input_model, ref_model, n_pix_grow_sat):
     return output_model
 
 
-def irs2_flag_saturation(input_model, ref_model, n_pix_grow_sat):
+def irs2_flag_saturation(input_model, ref_model, n_pix_grow_sat, use_readpatt):
```

Comment:
Add use_readpatt to parameters in docstring

Reply:
Done!